pipelines/winget-cli-restsource-ci-pr.yml

```diff
@@ -24,7 +24,11 @@ extends:
 
     settings:
       # Builds from forked repos do not have permissions to set tags
-      skipBuildTagsForGitHubPullRequests: true
+      skipBuildTagsForGitHubPullRequests: true
+
+    sdl:
+      binskim:
+        analyzeTargetGlob: '$(Build.SourcesDirectory)\**\Microsoft.WindowsPackageManager.Rest.dll;$(Build.SourcesDirectory)\**\Microsoft.WinGet.RestSource.dll;$(Build.SourcesDirectory)\**\Microsoft.WinGet.RestSource.Utils.dll;$(Build.SourcesDirectory)\**\Microsoft.WinGet.RestSource.AppConfig.dll;$(Build.SourcesDirectory)\**\Microsoft.WinGet.RestSource.Functions.dll;$(Build.SourcesDirectory)\**\Microsoft.WinGet.PowershellSupport.dll;'
```

Comment:
Could we change all the specific files for `Microsoft.WindowsPackageManager.*.dll` and `Microsoft.WinGet.*.dll`? It may catch other of our binaries from other projects, but these should pass binskim if they're being consumed here